eth-bytecode-db/eth-bytecode-db-server/tests/compiler_versions.rs

```diff
@@ -0,0 +1,113 @@
+mod verification_test_helpers;
+
+use crate::verification_test_helpers::{
+    smart_contract_verifer_mock::{
+        MockSolidityVerifierService, MockVyperVerifierService, SmartContractVerifierServer,
+    },
+    VerifierService,
+};
+use eth_bytecode_db_proto::blockscout::eth_bytecode_db::v2 as eth_bytecode_db_v2;
+use pretty_assertions::assert_eq;
+use rstest::rstest;
+use smart_contract_verifier_proto::blockscout::smart_contract_verifier::v2 as smart_contract_verifier_v2;
+use tonic::Response;
+
+async fn test_versions(
+    route: &str,
+    verifier: impl VerifierService<smart_contract_verifier_v2::ListCompilerVersionsResponse>,
+    verifier_response: smart_contract_verifier_v2::ListCompilerVersionsResponse,
+) {
+    let db_url = "sqlite::memory:";
```

Comment:
Just to understand: 
eth_bytecode_db can start with sqlite databae?

Reply:
It cannot in general. We cannot run migrations on sqlite. But for `list_compiler_versions` we do not need to initialize it, and thus sqlite is ok

Reply:
imo it's best to stick to single database (postgres)
we already have tests with it, so it isn't much harder to do

Reply:
@leviska Why, considering that `sqlite::memory` will not put any additional burden on the users?

Reply:
because they are different, something can work in one and not work in other
it also gives additional burden on developers, they might think "can this work on sqlite or do I use full postgres" without much reason to do so